lib/packages/crypto/src/converter.ts

```diff
@@ -306,4 +306,8 @@ export class Converter {
     public static convertTimestampToDate(timestamp: number) {
         return new Date(Date.UTC(2014, 7, 11, 2, 0, 0, 0) + timestamp * 1000);
     }
+
```

Comment:
I already extracted timestamp to date as isolated functions... this one will be extracted too